src/Whoops/Resources/views/frame_code.html.php

```diff
@@ -29,20 +29,6 @@
             $start = key($range) + 1;
             $code  = join("\n", $range);
         ?>
-            <style>
-             #frame-code-0 .linenums li:nth-child(<?= $line-$start ?>) {
-                 background-color: rgba(255, 100, 100, .07); 
```

Comment:
You probably did not mean to also remove background-color highlight from the lines.

 - [ ] Clarify the background-color highlight.